### PR TITLE
Reader: fix post and comment actions alignment

### LINKED ISSUE #95067: Reader repost action is misaligned
### Quick summary

The Repost action in the Reader post actions is misaligned compared to other actions.

Screenshot 2024-10-01 at 16 00 09

### Steps to reproduce

1. Go to https://wordpress.com/discover
2. Look at the post actions in the recommended posts stream
3. You should see the Repost action misaligned

### What you expected to happen

The post actions should all be aligned.

### What actually happened

The Repost icon and text are marginally out of alignment with the other post actions.

### Impact

All

### Available workarounds?

There is no user impact

### If the above answer is "Yes...", outline the workaround.

_No response_

### Platform (Simple and/or Atomic)

_No response_

### Logs or notes

_No response_

### PR DESCRIPTION
**Pull request by holdercp**
Fixes #95067

## Proposed Changes

* Fix alignment of Reader post and comment actions
* Make font size and spacing consistent across post and comment actions

| Before    | After |
| -------- | -------|
|Screenshot 2024-10-03 at 3 45 11 PM|Screenshot 2024-10-03 at 3 44 39 PM|
|Screenshot 2024-10-03 at 3 43 26 PM|Screenshot 2024-10-03 at 3 43 49 PM|

## Why are these changes being made?

* The post and comment actions were subtlety misaligned #sweatthedetails
* There were various manual position adjustments of the icons/labels that were unnecessary after resetting back to base styles, resulting in less CSS overrides

## Testing Instructions

* Navigate to `/discover`, `/read`, or any Reader post feed.
* Check the following: 
  * Alignment of the post actions on the post card
  * Alignment of the comment actions on the post card
  * Alignment of the post actions on the full post view
  * Alignment of the comment actions on the full post view
  * Alignment of the post actions in the full view sidebar
  * Check the relevant UI above on the mobile breakpoint

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/blocks/comment-button/style.scss [hunk 1]
```diff
@@ -21,23 +21,11 @@
...
-		font-weight: 500;
-	}
 }
 
-.reader-full-post .reader-post-actions .comment-button__label {
-	/* stylelint-disable-next-line declaration-property-unit-allowed-list */
-	font-size: 13px;
-	font-weight: 500;
+.reader-full-post .reader-post-actions .comment-button .comment-button__label {
+	font-size: rem(14px);
```
Comment (holdercp) [inline client/blocks/comment-button/style.scss:30]: The other actions were set at 14px font size, so I made this consistent.

File: client/blocks/comments/comment-actions.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
 	gap: 22px;
 	justify-content: flex-start;
 	list-style: none;
+	margin-top: 4px;
```
Comment (holdercp) [inline client/blocks/comments/comment-actions.scss:10]: We were implicitly getting extra height from line height in the action items (which was in part causing the alignment issue). After resetting line height back to a baseline, the comment actions felt too tight against the comment, so this adds back that space explicitly.

File: client/blocks/like-button/style.scss [hunk 3]
```diff
@@ -64,19 +64,22 @@
...
+}
+
+.reader-full-post {
+	.reader-post-actions .like-button {
+		padding: 0;
+
+		.like-button__label {
+			font-size: rem(14px);
+		}
+	}
```
Comment (holdercp) [inline client/blocks/like-button/style.scss:84]: Consolidating here and setting the font size to match the other actions.

File: client/blocks/reader-full-post/style.scss [hunk 4]
```diff
@@ -131,15 +131,6 @@ $reader-full-post-desktop-min: $reader-full-post-sidebar-width + $reader-full-po
 		font-size: $font-body;
 		line-height: 24px;
 	}
-
-	.comment-button__label {
```
Comment (holdercp) [inline client/blocks/reader-full-post/style.scss:135]: After working with the line height and making sure font sizes and such were consistent, flexbox took care of the alignment so these manual adjustments were unnecessary.

**Review by davemart-in — APPROVED** [review]
Changes look solid to me. I tested a couple of different feeds and resolutions. 

Love seeing all of that red in the diff! :-)

**Review by Addison-Stavlo — APPROVED** [review]
This looks great to me as well, love the cleanup!

Some before and after from different areas of the reader that this affects:

BEFORE (full post page article footer)
Screenshot 2024-10-03 at 5 04 24 PM

AFTER
Screenshot 2024-10-03 at 5 04 30 PM

BEFORE (full post page author section)
Screenshot 2024-10-03 at 5 04 47 PM

AFTER
Screenshot 2024-10-03 at 5 04 51 PM

BEFORE (stream cards - compact)
Screenshot 2024-10-03 at 5 05 10 PM

AFTER
Screenshot 2024-10-03 at 5 05 15 PM

BEFORE (stream card - standard)
Screenshot 2024-10-03 at 5 09 02 PM


AFTER
Screenshot 2024-10-03 at 5 09 08 PM

**Pull request merged**